src/components/ToastDrawer/ToastDrawer.scss

```diff
@@ -4,3 +4,24 @@
   z-index: 100;
   padding-right: 40px;
 }
+
+.toast-header {
+  @include themed {
+    background-color: rgba(t("toast-bg"), 0.85);
```

Comment:
@JosefBud Thoughts on partially transparent toasts? I'm not sure I like the idea of losing contrast ratio just for the sake of being able to see behind it.

Reply:
Note I tried to stay as close as possible to the current styling rules as possible, these were just implicit rules.

Reply:
Can you give some examples of why it should be transparent? I just don't know why you'd need to be able to see through them when they're only up for a couple of seconds.

Reply:
What I meant is that they were transparent already before and I recreated this transparency explicitly, because I didn't want to change the current style. At least that's how it was displayed on my system.
I will remove this then, because I don't see a good reason for them being transparent either.

Reply:
Sounds good, sorry for the confusion!